backend/copr_backend/rpm_builds.py

```diff
@@ -133,6 +137,18 @@ def predicate(x):
         super().__init__(predicate, limit, name="arch_{}".format(architecture))
 
 
+class ArchitectureUserWorkerLimit(GroupWorkerLimit):
```

Comment:
There is probably nothing we can do but the inheritance doesn't look right (UserWorkerLimit from GroupWorkerLimit)

Reply:
Yeah, I think we could rename `GroupWorkerLimit` into `HashedWorkerLimit` :-)?